ctapipe/reco/shower_processor.py

```diff
@@ -69,3 +72,18 @@ def __call__(self, event: ArrayEventContainer):
         """
         k = self.reconstructor_type
         event.dl2.stereo.geometry[k] = self.reconstructor(event)
+
+        # compute and store the impact parameter for each reconstruction (for
+        # now there is only one, but in the future this should be a loop over
+        # reconstructors)
+
+        # for the stereo reconstructor:
+        impact_distances = shower_impact_distance(
+            shower_geom=event.dl2.stereo.geometry[k], subarray=self.subarray
+        )
+
+        for tel_index, impact_distance in enumerate(impact_distances):
+            tel_id = self.subarray.tel_ids[tel_index]
+            event.dl2.tel[tel_id].impact[k] = TelescopeImpactParameterContainer(
+                distance=impact_distance
```

Comment:
If we don't fill the uncertainty here, why do we define it at all?

Reply:
More for future use and to remind us to implement i - we have a requirement to estimate uncertainty in reconstructed quantities when possible.  This would just be a propagation of the uncertainty in the core reconstruction, which is also not currently implemented yet.